.github/ISSUE_TEMPLATE/REPORT_A_BUG.yml

```
@@ -0,0 +1,84 @@
+name: '🐛 Report a bug'
+description: 'Is something not working as you expect?'
+labels: []
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thank you for taking the time to file a bug report!
+
+        Before posting, please check that the bug hasn't already been:
+        * [fixed in the next release](https://github.com/stylelint/stylelint/blob/main/CHANGELOG.md)
+        * [discussed previously](https://github.com/stylelint/stylelint/search)
```

```suggestion
        * [discussed previously](https://github.com/search/advanced?r=stylelint%2Fstylelint&type=Issues)
```

The page at the previous link must of changed at some point as it's not very useful now. This new one has a search bar at the top (and will search only issues).

---

Looks good!

FYI. I think <https://github.com/stylelint/stylelint/search?type=issues> is also good. Anyway, the search link can be improved. 😄